### PR TITLE
Add TabsModify Demo

### PR DESCRIPTION
**Pull request by TimChen44**
Added an example of using an array to display RadzenTabsItems, and provided examples of adding and removing RadzenTabsItems.

Question: The Tab component provides the bind SelectedIndex attribute, but both AddTab and RemoveItem operations on TabItem require the RadzenTabsItem type to be passed in. Can you provide the bind SelectedTabsItem property or can you use SelectedIndex to obtain the SelectedTabsItem method?

### CODE REVIEW

File: RadzenBlazorDemos/Pages/TabsPage.razor [hunk 1]
```diff
@@ -33,3 +33,14 @@
...
 </RadzenExample>
+
+
+<RadzenText Anchor="tabs#client-render-mode" TextStyle="TextStyle.H5" TagName="TagName.H2" class="rz-pt-12">
+    TabItems modify
+</RadzenText>
+<RadzenText TextStyle="TextStyle.Body1" class="rz-mb-8">
+    Demonstrating modifications to the TabItem collection
+</RadzenText>
+<RadzenExample ComponentName="Tabs" Example="TabsClient">
```
Comment (enchev) [inline RadzenBlazorDemos/Pages/TabsPage.razor:44]: Should point to this example component/page not to TabsClient.

File: RadzenBlazorDemos/Pages/TabsPage.razor [hunk 2]
```diff
@@ -33,3 +33,14 @@
 <RadzenExample ComponentName="Tabs" Example="TabsClient">
     <TabsClient />
 </RadzenExample>
+
+
+<RadzenText Anchor="tabs#client-render-mode" TextStyle="TextStyle.H5" TagName="TagName.H2" class="rz-pt-12">
```
Comment (enchev) [inline RadzenBlazorDemos/Pages/TabsPage.razor:38]: The anchor is wrong as well.

**Comment by enchev** [comment]
Although AddTab() and RemoveItem() methods are public they should be avoided.

**Comment by TimChen44** [comment]
Adjusted as required.

Is there a plan to add the Remove Tab button on the RadzenTabs tab?

**Comment by enchev** [comment]
> Adjusted as required.
> 
> Is there a plan to add the Remove Tab button on the RadzenTabs tab?

We don't have such plans.

**Pull request merged**